i18n/en/docusaurus-plugin-content-docs/current/reference/units/layers/index.md

```diff
@@ -15,7 +15,7 @@ The first level of application partitioning, according to the **scope of influen
     ├── app/                    # Initializing application logic
     ├── processes/              # (Optional) Application processes running over pages
     ├── pages/                  # Application pages
-    ├── widgets/                # Independent and self-contained blocks for pages
+    ├── widgets/                # Applets intended to be used within pages, including the app layout
```

Comment:
issue: not a big fan of the term _applets_ because it is not very common, thus open to multiple interpretations. To me it indicates that a widget must be almost a complete usecase packed in a little box in the application.

suggestion: 
```suggestion
    ├── widgets/                # Self-contained blocks to be used within pages, including the app layout
```

Reply:
Hmm, I'll try to think a better definition then

Reply:
```suggestion
    ├── widgets/                # Compositions of lower-level modules providing complex user experience as self-sufficient UI blocks and intended to be rendered on pages
```